src/lib/api/api.go

```diff
@@ -119,7 +120,7 @@ func (a *API) Home(w http.ResponseWriter, r *http.Request) {
 	msg := "ping from " + r.RemoteAddr + ", sent pong in monitor"
 	log.Infoln(msg)
 	a.Data = msg
-	a.send(w, a.marshalled())
+	a.send(w, a.Data, nil)
```

Comment:
Can you remove a.Data and simply pass Data? The dependency on a should never be there. Thats what the race is all about.

Reply:
Yeah sure